graviti/portex/base.py

```diff
@@ -235,6 +247,63 @@ def copy(self: _T) -> _T:
         return deepcopy(self)
 
 
+class PortexRecordBase(
+    PortexType, UserMutableMapping[str, PortexType]
+):  # pylint: disable=abstract-method
```

Comment:
why disable abstract-method?

Reply:
Because `_from_pyarrow` is not implemented.

Reply:
Because `_from_pyarrow` is not implemented.